introduction.md

```diff
@@ -0,0 +1 @@
+I don't really know how to code in Ruby, but I plan on learning soon.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.